examples/demo/world_map.py

```diff
@@ -34,7 +38,7 @@ class WorldMapPlot(HasTraits):
 
     # The URL which points to the world map image to be downloaded
     image_url = Str(
-        "http://eoimages.gsfc.nasa.gov/ve//2433/land_shallow_topo_2048.jpg"
+        "https://upload.wikimedia.org/wikipedia/commons/9/91/Winkel_triple_projection_SW.jpg"
```

Comment:
This uses a different projection, and so the plotted points are not correct (eg. Austin will not be plotted correctly).

The original image is this one in Wikimedia: https://commons.wikimedia.org/wiki/File:Land_shallow_topo_2048.jpg

So this URL may be the right one to use (although it is very high resolution): https://upload.wikimedia.org/wikipedia/commons/9/91/Land_shallow_topo_2048.jpg

Reply:
And this is the source from NASA: https://visibleearth.nasa.gov/images/57752/blue-marble-land-surface-shallow-water-and-shaded-topography/57754l

So this may be even better: https://eoimages.gsfc.nasa.gov/images/imagerecords/57000/57752/land_shallow_topo_2048.jpg

Reply:
Thanks, just changed to "https://eoimages.gsfc.nasa.gov/images/imagerecords/57000/57752/land_shallow_topo_2048.jpg" as you proposed. I assume we don't need a copyright header since it's the same NASA image as before?